test/acceptance/tests/controller/controller_test.go

```diff
@@ -92,6 +92,10 @@ func TestController(t *testing.T) {
 					proxyDefaultEntry, ok := entry.(*api.ProxyConfigEntry)
 					require.True(r, ok, "could not cast to ProxyConfigEntry")
 					require.Equal(r, api.MeshGatewayModeLocal, proxyDefaultEntry.MeshGateway.Mode)
+					require.Equal(r, "tcp", proxyDefaultEntry.Config["protocol"])
```

Comment:
let's also test `foo` and `member` for completeness. Or we can remove those from the fixture. I like testing multiple types, so maybe bool, array, map, string?

Reply:
Hmm, good call. I think I've made an assumption that these are going to be ignored by consul, but they're not!

Reply:
done!